docs/how-to/configure-denylist.md

```diff
@@ -1,24 +0,0 @@
-# How to restrict self-hosted runner network access
```

Comment:
I am fine with deleting, but I think we could have a short note in the howto mentioning that its recommended to allow traffic from the runner vm's to the juju machine charm model.

Reply:
This how-to applies to the denylist, that is not used in openstack. What do you mean by runner vm to the juju machine charm model?

Reply:
What I meant was that we could add a note to the openstack howto that it is recommended to forbid outgoing traffic from the runner vm to the network hosting the juju charm units. The howto currently only mentions that the charm needs access to the runner vm, but does not talk about disallowing traffic in the other direction: 

>Note that the network should be configured to allow traffic from the charm deployment (Juju machine) to the OpenStack virtual machine, and traffic from the OpenStack virtual machine to GitHub.

But as this is not strictly related to this PR, we could add this in a follow-up.

Reply:
Added here to not forget: https://warthogs.atlassian.net/browse/ISD-2856